frontend/app/routes.json

```diff
@@ -426,6 +426,20 @@
               }
             ]
           },
+          {
+            "id": "$lang+/_protected+/status-check+/_route",
+            "file": "routes/$lang+/_protected+/status-check+/_route.tsx",
+            "children": [
+              {
+                "id": "$lang+/_protected+/status-check+/index",
+                "file": "routes/$lang+/_protected+/status-check+/index.tsx",
+                "paths": {
+                  "en": "/:lang/status-check",
+                  "fr": "/:lang/statut"
```

Comment:
`/fr/statut` already exists for the French public-facing Status Checker. We need to define a new path for the protected Status Checker otherwise when we toggle language, we end up on the public-facing one.